app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class UsersController < ApiController
+      def rents_index
+        render json: Rent.all, each_serializer: UserRentSerializer
+      end
+
+      def rents_params
```

Comment:
As a convention, `*_params` methods are always private because nobody will invoke them from outside the controller